fabric-fluid-v1/src/main/java/net/fabricmc/fabric/impl/fluid/BucketItemAccessor.java

```diff
@@ -0,0 +1,7 @@
+package net.fabricmc.fabric.impl.fluid;
+
+import net.minecraft.fluid.Fluid;
+
+public interface BucketItemAccessor {
+	Fluid getFluid();
```

Comment:
make `@Accessor` please :1st_place_medal:

Reply:
no, there's a BucketItemMixin, why would u mixin to a class twice

Reply:
It's completely normal if you need to both access and mixin the same class.

Reply:
does it really matter though, and also mixing into it twice doesn't achieve anything other than delaying startup time a little bit

Reply:
why are you asking for conflicts

Reply:
at least prefix the method with fabric_ if you hate accessor so much

Reply:
oh yea I forgot about those, i'll rename it :tiny_potato: